docs/source/api/entities.md

```diff
@@ -0,0 +1,329 @@
+---
+title: "PIM Connector Framework"
+description: Import Data from any PIM
+---
+
+To upload huge amount of data to Unchained, for example if you pump the data from a PIM or ERP system, we have created a special API.
+
+## Bulk Import API
+
+Using the Bulk API stores the data in the Unchained Work Queue before processing and enables some neat features:
+
+1. Cloud Native: Allows dedicated background or worker instances to do the actual processing
+2. Transparent Process: Also stores the result on the work item, making successful and failed imports queryable.
+3. Captures Sync issues and reports those with a Status E-Mail to a central E-Mail Address (Sync error reporting)
+4. Performance: Only downloads assets that are not downloaded yet and tries to optimize the actual write calls to the database by using MongoDB bulk operations, making sync processing fast
+5. Push-based: Immediate representation of changes
+
+In some situations, it's propably wise to develop a sync microservice: You have a source system that
+- generates "Pull-based" data feeds, or
+- can not adopt to the JSON described below or
+- needs to merge data from different systems
+
+### Endpoint
+
+The main entry point to send import events is:
+Mutation.sendBulkImportEvents. The mutation takes an array of events that consist out of a type, an operation and a payload.
+
+Supported entity types:
+- PRODUCT
+- ASSORTMENT
+- FILTER
```

Comment:
Should we also provide the possibility to import users, orders and other stuff?

Reply:
For some cases it might be beneficial to additionally import users, orders, reviews and subscriptions. Especially if somebody migrates from an existing webshop. I will add it and specify that this will be v2 of the Import API